src/components/CharacterDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { useQuery } from "@apollo/client";
+import { GET_CHARACTER } from "../graphql/Characters";
+import CharacterDetailsCard from "./CharacterDetailsCard";
+import { CharacterDetailsProps } from "../utils/types";
+import Loading from "./Loading";
+
+const CharacterDetails: React.FC<CharacterDetailsProps> = ({
```

Comment:
Avoid the React.FC usage is not worth it even though now it's usage is safety, I would define this in the next way

const CharacterDetails = ({ selectedCharacter} : CharacterDetailsProps) =>

Reply:
ok I will take into account that.